### PR TITLE
completed TODO

### PR DESCRIPTION
**Pull request by DavidRomanovizc**
also I refactored the code

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes # Todo 

## Type of change

Please delete options that are not relevant.

- [x] Documentation (typos, code examples or any documentation update)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test 

**Test Configuration**:
* Windows 10
* Python version: 3.9+ 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by wydilya** [comment]
lgtm

**Review by wydilya — APPROVED** [review]
lgtm!

File: examples/advanced_executor_example.py [hunk 1]
```diff
@@ -20,19 +20,19 @@
...
 import ssl
 import sys
 
-from aiogram import Bot
-from aiogram.dispatcher import Dispatcher
-from aiogram.dispatcher.webhook import *
-from aiogram.utils.executor import start_polling, start_webhook
-
 logging.basicConfig(level=logging.INFO)
+API_TOKEN = 'API_TOKEN_HERE'
```
Comment (Olegt0rr) [inline examples/advanced_executor_example.py:33]: You remove this 
```
# TODO: Move token to environment variables.
```

And add this
```
API_TOKEN = 'API_TOKEN_HERE'
```

Looks weird

File: examples/advanced_executor_example.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ async def on_startup(dispatcher, url=None, cert=None):
         await bot.delete_webhook()
 
 
-async def on_shutdown(dispatcher):
+async def on_shutdown():
```
Comment (Olegt0rr) [inline examples/advanced_executor_example.py:85]: Why do you remove passed argument?

File: examples/advanced_executor_example.py [hunk 3]
```diff
@@ -51,15 +51,15 @@ async def cmd_start(message: types.Message):
     return SendMessage(message.chat.id, f"Hello, {message.from_user.full_name}!")
 
 
-def setup_handlers(dispatcher: Dispatcher):
+def setup_handlers(dp: Dispatcher):
```
Comment (Olegt0rr) [inline examples/advanced_executor_example.py:54]: Why you do this renaming? `dp` name already used by you in global scope...

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: examples/advanced_executor_example.py (hunk 2 above)
Comment (DavidRomanovizc) [reply examples/advanced_executor_example.py:85]: > Почему вы удаляете переданный аргумент?
потому что он нигде не используется

Comment (Olegt0rr) [reply examples/advanced_executor_example.py:85]: But dispatcher is passed here :)

File: examples/advanced_executor_example.py [hunk 4]
```diff
@@ -20,19 +20,19 @@
...
 
-from aiogram import Bot
-from aiogram.dispatcher import Dispatcher
-from aiogram.dispatcher.webhook import *
-from aiogram.utils.executor import start_polling, start_webhook
-
 logging.basicConfig(level=logging.INFO)
+API_TOKEN = 'API_TOKEN_HERE'
+bot = Bot(token=API_TOKEN)
+dp = Dispatcher(bot=bot)
```
Comment (Olegt0rr) [inline examples/advanced_executor_example.py:35]: Duplicated from `main`

**Pull request closed without merging**